components/navigation/navigation.jsx

```diff
@@ -93,22 +91,22 @@ export default class Navigation extends React.Component {
           <Link
             className="navigation__icon"
             title="GitHub Repository"
-            to="//github.com/webpack/webpack">
+          to="//github.com/webpack/webpack">
```

Comment:
Yeah, I fixed it. Strange thing is that my editor does favor editorconfig, but for an obscure reason prefers to format those fragments the way they were in the source (and doesn't even let me edit them back). Some misbehaving Atom package I guess.